Assets/Scripts/Calendar UI.cs

```diff
@@ -80,46 +102,114 @@ private void HandleBackwardClick()
         SetNumberOfBlanks();
         PopulateCalendarGrid();
     }
+    private void HandleCellImageSelect(Cell cell)
+    {
+        foreach (DayCell c in populationList)
```

Comment:
no need to change this - you can auto resolve as its not bad here but I'd really get used to using for loops instead of foreach loops when you can. Foreach is prettier but significantly more limited and (I think) mostly used for dealing with pure IEnumerable collections (which we can talk about). 

Foreach isn't bad by itself when it works (like here) but it can be a pain to rewrite -> for loop when you need to.